app/config.js

```diff
@@ -15,21 +15,31 @@ define(function (require) {
 		/* Time limit for learning sessions in seconds */
 		TIME_LIMIT: 600,
 		/* Time before the next item is presented while learning */
-		FEEDBACK_DELAY_CORRECT: 0.5,
+		FEEDBACK_DELAY_CORRECT: 1,
 		FEEDBACK_DELAY_INCORRECT: 10,
 		/* Path to user database */
-		DATABASE_USER: __dirname + "/database/user.sqlite",
+		DATABASE_USER: "/database/user.sqlite",
 		/* Standard database when no user database is present */
-		DATABASE_SLIMSTAMPEN: __dirname + "/database/slimstampen.sqlite",
+		DATABASE_SLIMSTAMPEN: "/database/slimstampen.sqlite",
 		ALGORITHM: "flashcard", // flashcard; slimstampen;
-		LANGUAGE: "nl",
+		LANGUAGE: "en",
 		ONLINE_HOST: "db4free.net",
 		ONLINE_USER: "enricdz148_slim",
 		ONLINE_DATABASE: "enricdz148_slim",
-		ONLINE_PASSWORD: "exVhqZJas"
+		ONLINE_PASSWORD: "exVhqZJas",
+		MESSAGES: "#messages",
+		ERRORS: "#errors"
 	};
 
-
+	// Fix for compiled program of database
+	var dir = __dirname;
```

Comment:
I have no clue what is happening here. Some descriptive function names would be nice, both in the body and the guard of the if-else-statement.